src/emulator.cpp

```diff
@@ -256,6 +256,14 @@ void Emulator::Run(const std::filesystem::path& file) {
     std::exit(0);
 }
 
+void Emulator::Run(int& argc, char* argv[]) {
```

Comment:
Why is `argc` a reference?

Reply:
Good question, I don't know. I'll pass by value instead